2022-11-13 21:09:23.979 | INFO     | __main__:main:91 - GuessNumberGame Server Start Successfully.
2022-11-13 21:09:32.193 | DEBUG    | __main__:route_ngg:22 - conn:<websockets.legacy.server.WebSocketServerProtocol object at 0x7fa283c5d730>
2022-11-13 21:09:34.074 | INFO     | __main__:route_ngg:32 - got a message:{"api":"create_room", "data":{"username":"zxz_test1","roomname":"æˆ‘1"}}
2022-11-13 21:09:34.077 | ERROR    | __main__:route_ngg:43 - [main.route_ngg] remote_address('101.64.140.35', 63864) errpr=Traceback (most recent call last):
  File "/root/code/pythoncode/PYProject/GuessNumberGame/src/main.py", line 36, in route_ngg
    res = handler(data_d["data"])
  File "/root/code/pythoncode/PYProject/GuessNumberGame/src/NGGService.py", line 30, in createRoom
    roomid = roomCRUD.insert_without_id(RoomPO.bo2po())
TypeError: bo2po() missing 1 required positional argument: 'bo'

